### ISSUE TITLE
Add underline to H3 link (and others?) styling for "Helpful links" per a11y recommendation

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/vets-design-system-documentation#157]
# Feature Request

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## Is this feature request relating to an existing component or utility? Please describe.

- Component/utility name: H3 link (used in "Helpful links" content section as seen at the bottom of https://www.va.gov/change-address/ )

I am creating a design spec via https://github.com/department-of-veterans-affairs/va.gov-team/issues/2691 and I am including a similar section as "Helpful links." 

I asked @1Copenut aka Trevor for an a11y review. He says that the H3 link header should be underlined:

The helpful links would benefit from an underline or other visual indicator. The links rely on color to distinguish them from non-interactive text. See https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/F73 for further description of this item.

I'm not sure if this is a change to the standard component or a utility class.

**Comment by humancompanion-usds** [comment]
I believe this is fixed in the [Link - Collections](https://design.va.gov/components/link/collection#examples) we have and it appears that Related Links is now used on the change address page. 
![Screen Shot 2022-06-16 at 5 17 01 PM](https://user-images.githubusercontent.com/52176351/174165904-c9dc8718-ab29-4246-9688-acdeff61af46.png)

@artsymartha68 - I believe this to be fixed so I'm closing this but if you disagree please do reopen.

**Comment by artsymartha68** [comment]
Agreed this is fixed, thanks!
